web/themes/contrib/civictheme/includes/form.inc

```diff
@@ -61,6 +61,11 @@ function civictheme_preprocess_form_element(array &$variables): void {
   _civictheme_preprocess_form__theme($variables);
   _civictheme_preprocess_form_element__classes($variables);
   _civictheme_preprocess_form_element__description($variables);
+
+  // Unset checkbox and radio label.
+  if (isset($variables['type']) && in_array($variables['type'], ['checkbox', 'radio'])) {
+    unset($variables['label']);
```

Comment:
Unset the form element label from the form element and render in checkbox and twig template.